CHANGELOG-DCA.rst

```diff
@@ -2,6 +2,19 @@
 Release Notes
 =============
 
+.. _Release Notes_7.50.2:
+
+7.50.2 / 6.50.2
+======
+
+.. _Release Notes_7.50.2_Bug Fixes:
+
+Bug Fixes
+---------
+
+- Fixes a bug introduced in `7.50.0` preventing `DD_TAGS` to be added to `kubernetes_state.*` metrics.
```

Comment:
wasn't this in 7.50.1 ?

Reply:
yes

Reply:
I see that his change is in 7.50.1 Agent changelog but no in DCA one. What should be the correct state?

Reply:
I see now, that changelog entry was added later and probably it was incorrectly caught by the tooling. I can change the DCA changelog entry to 7.50.1 to make it consistent. Would that be ok?